client/src/components/postcall/RejoinLink.tsx

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { Link } from '@fluentui/react';
+
+export interface RejoinLinkProps {
+  onRejoinCall: () => void;
+}
+
+export const RejoinLink: React.FunctionComponent<RejoinLinkProps> = (props: RejoinLinkProps) => {
+  return (
+    <Link
+      onClick={() => {
```

Comment:
Does this work?

```
onClick={props.onRejoinCall}
```

Reply:
Thanks, this works, have mad this change!